### PR TITLE
desktop-version for my portfolio project #4

### PR DESCRIPTION
**Pull request by Abass-rashid**
1.  used media query for responsiveness and to be desktop version.
 2. no linters error
 3. flowed correct Github flow
 4. used flexbox for page layout
 5. the project is responsive for both mobile and desktop version
 6. added nav links in case of the mobile version 
 7. used min-width of 768px

### CODE REVIEW

**Comment by Abass-rashid** [comment]
I have made a change on the way recommended please take your time to review it again thanks.

File: index.html [hunk 1]
```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
...
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
```
Comment (youmari) [inline index.html:113]: your card doesn't have the same padding as the one in the Figma design please try to change your card to be the same as the one in the Figma design as the previous reviewer asked

- [ ] Your card
image

- [ ] Figma design 
image

File: index.html [hunk 2]
```diff
@@ -145,6 +208,7 @@ <h2 class="form-header">Contact Me</h2>
                    </label>
                 <textarea name="message" id="msg" class="text-area" placeholder="Write your Message here" maxlength="500" required></textarea>
             <button type="submit">Get in touch</button>
```
Comment (youmari) [inline index.html:210]: - [ ] [OPTIONAL] your contact form in the desktop version doesn't look like the one in the Figma design, I suggest sticking to the Figma design and making some changes so your contact form will be as one in the Figma design 

- [ ] Yours
image

- [ ] Figma design
image

File: index.html [hunk 3]
```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
...
+                  <h3><span>languages</span><img src="images/arr-down.svg" alt=""></h3>
+                  <ul class="stack">
+                    <li><img src="images/js.svg" alt="">&nbsp;&nbsp;&nbsp;<span>JavaScript</span></li>
+                    <li><img src="images/html.svg" alt="">&nbsp;&nbsp;&nbsp;<span>HTML</span></li>
+                    <li><img src="images/css.svg" alt="">&nbsp;&nbsp;&nbsp;<span>CSS</span></li>
+                  </ul>
+                  <h3><span>frameworks</span><img src="images/arr-right.svg" alt=""></h3>
+                  <h3 class="skills"><span>skills</span><img src="images/arr-right.svg" alt=""></h3>
+                  </div>
+                </div>
```
Comment (youmari) [inline index.html:193]: - [ ] [OPTIONAL] Also, your about section in the desktop version doesn't look like the one in the Figma design, I suggest sticking to the Figma design and making some changes so your contact form will be as one in the Figma design 

- [ ] Yours 
image

- [ ] Figma design 
image

File: index.html [hunk 4]
```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
...
+                  </div>
+                  
+                </article>
+                <!--third card-->
+                <article class="card">
+          
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-3.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>multi-post stories</h3>
```
Comment (youmari) [inline index.html:145]: - [ ] [OPTIONAL] PLEASE it would be better if you use the right font provided in the Figma design

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Abass-rashid , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 5]
```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
...
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
```
Comment (ShahierNashaat) [inline index.html:113]: - [ ] [Optional] Kindly be noticed that the project cards in your mobile version don't look good please fix that.
Screenshot from 2022-04-28 21-00-04

**Review by ShahierNashaat — APPROVED** [review]
Hi @Abass-rashid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**